finances.js

```diff
@@ -73,17 +73,26 @@ window.onload = function () {
     renderTransactions(transactions);
 }
 
+function addOrUpdate(event) {
+    let type = document.getElementById("submitBtn").textContent;
```

Comment:
depending on if it's 'Add' or 'Update' we call the correct function. We get the trID from the form each time so we don't have issues with hardcoded values being used repeatedly